src/identity.cc

```diff
@@ -27,6 +27,7 @@
 #include <map>
 #include <memory>
 #include <thread>
+
```

Comment:
Don't add this line

Reply:
Fixed. PS: the perf numbers are also updated on kibana/CI and they are nearly up to the legacy custom backend performance.